Configuration API (#4913)
DataDog/documentation: content/en/api/logs_pipelines/logs_pipelines.md

---
title: Logs Pipelines
type: apicontent
order: 24
external_redirect: /api/#logs-pipelines
---

## Logs Pipelines

Pipelines and processors operate on incoming logs, parsing and transforming them into structured attributes for easier querying.

> * See the [Pipelines Configuration Page][1] to see the pipelines and processors currently configured in our UI.

Reviewer: ```suggestion
* See the [Pipelines Configuration Page][1] for a list of the pipelines and processors currently configured in our UI.
```

> * See the [Pipeline documentation][2] to learn more

Reviewer: ```suggestion
* For more information about Pipelines, see the [Pipeline documentation][2].
```
